scripts/wildcard-domain-processor/wildcard-domains-updater.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-await-in-loop,no-restricted-syntax,no-console */
+import * as path from 'path';
+import { RuleParser } from '@adguard/agtree';
+// There is no type definition available for the following import.
+// @ts-ignore
+import { findDeadDomains } from '@adguard/dead-domains-linter/src/urlfilter';
+import { extractRuleDomains } from './domain-extractor';
+import { utils } from './utils';
+import { TOP_LEVEL_DOMAIN_LIST } from './top-tld';
+import { findFilterFiles, readFile, writeFile } from './file-utils';
+
+/**
+ * Parses a rule and extracts domains from it.
+ * @param rule - The rule to extract domains from.
```

Comment:
```suggestion
 * @param rule The rule to extract domains from.
```

Reply:
There are a lot of such cases, they can be improved later